### DISCUSSION TITLE
Welcome to Quetzalcoatl Discussions!

### DISCUSSION THREAD

**Discussion by RickTaylor79** [publicsamples/Quetzalcoatl#2] [Announcements]
Ask questions, make suggestions.. Thank kind of thing :)

**Comment by HushSoundz** [comment]
how do you install this? please!

**Reply by RickTaylor79** [reply]
Take a look at this [video](https://www.youtube.com/watch?v=XPiBUeKSPFc&t=75s), or instructions [here](https://github.com/publicsamples/Quetzalcoatl/wiki/Installation-v0.6.5).

**Comment by HushSoundz** [comment]
Thanks alot! This helped.
